Golem.Tests/Utils.cs

```diff
@@ -152,11 +218,37 @@ public async Task InitializeAsync()
             _requestorAppKey = _requestor.getTestAppKey();
         }
 
+        public async Task StartGolem(IGolem golem, String golemPath, ChannelReader<GolemStatus> statusChannel)
+        {
+            _logger.LogInformation("Starting Golem");
+            var startTask = golem.Start();
+            Assert.Equal(GolemStatus.Starting, await ReadChannel(statusChannel));
+            await startTask;
+            Assert.Equal(GolemStatus.Ready, await ReadChannel(statusChannel));
+            var providerPidFile = Path.Combine(golemPath, "modules/golem-data/provider/ya-provider.pid");
+            await TestUtils.WaitForFile(providerPidFile);
```

Comment:
I don't think test should wait for this file, rather it should check it's existence. Note that Start should finish when yagna is ready to receive REST calls, if we don't have pid file until this moment, that indicates problem.

Reply:
I removed assertions for `.log.gz` because waiting for logger to archive file results with flaky tests.